.github/workflows/component.yml

```diff
@@ -96,13 +104,15 @@ jobs:
 
       - name: Integration testing
         run: |
-          if [[ "${COMPONENT_DIR}" == "interactive_ai/workflows/geti_domain/project_ie" ]]; then
+          if [[ "${{ steps.component-config.outputs.integration_builder }}" == "false" ]]; then
```

Comment:
In the [UI's Makefile](https://github.com/open-edge-platform/geti/blob/main/web_ui/Makefile#L28) we overwrite the builder step like,
```
builder:
	@echo "Not applicable"
```

This allows us to skip / ignore these steps from the component configuration, instead of the CI configuration.
Could we do something similar here (allowing us to remove this if statement)? So instead of doing an `@echo "Not applicable"` the target could run the provided arguments instead of forwarding them to a docker builder?